tests/cypress/e2e/unit_tests/upgrade.spec.ts

```diff
@@ -50,21 +50,30 @@ describe('Upgrade tests', () => {
       if ( elementalUIVersion == '1.0.0' ) {
         cy.exec(`sed -i 's/# namespace: fleet-default/namespace: fleet-default/g' assets/managedOSVersionChannel.yaml`);
         cy.exec(`sed -i 's@image: %UPGRADE_CHANNEL_LIST%@image: ${upgradeChannelList}@g' assets/managedOSVersionChannel.yaml`);
```

Comment:
Again for later maybe: use var/const for the filename?

Reply:
not needed, it will be removed once we will get a new stable.